### PR TITLE
Add not found page

### PR DESCRIPTION
**Pull request by Alhajideen**
In this milestone, I have added the folllowing;
- Add a not-found page for wrong urls
Purpose
- This proactive approach takes users to a customized page, maintaining app structure instead of the default react 404 page

NB: Design is responsive across all devices

### CODE REVIEW

**Comment by kungfuchicken** [comment]
It looks like there is not an open issue on this that you're resolving. I'm just confirming, this is just a general improvement you're suggesting?

**Comment by Alhajideen** [comment]
Positive. There is no open issue for this. I am suggesting a general improvement.

**Review by mohamdlog — CHANGES_REQUESTED** [review]
**Requests:**
- Please change message to: "The page you are trying to open does not exist. You may have mistyped the address, or the page may have been moved to a different URL. If you believe this is an error, please contact support."
- The font differences between the files are due to different styling methods. Please ensure the font on this page matches the others for consistency.

Otherwise, looks good.

File: frontend/src/App.js [hunk 1]
```diff
@@ -4,12 +4,12 @@ import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
...
-import ReportDashboard from './components/ReportDashboard'; 
-import LandingPage from './LandingPage'; 
-import AboutUs from './components/AboutUs'; 
+import ReportDashboard from './components/ReportDashboard';
+import LandingPage from './LandingPage';
+import AboutUs from './components/AboutUs';
 import NavigationBar from './pages/NavigationBar';
-import "./styles.css"
-
+import './styles.css';
```
Comment (mohamdlog) [inline frontend/src/App.js:11]: Good for consistency

File: frontend/src/App.js [hunk 2]
```diff
@@ -18,16 +18,16 @@ function App() {
...
-          <Route exact path='/dashboard' element={<FuzzyDashboard />} /> 
-          <Route exact path = '/report-dashboard' element = {<ReportDashboard/>} />  
-          <Route exact path = '/' element = {<LandingPage/>} />  
-          <Route exact path = '/about-us' element = {<AboutUs/>} />  
-          
+          <Route exact path='/simulation' element={<Wizard />} />
+          <Route exact path='/dashboard' element={<FuzzyDashboard />} />
+          <Route exact path='/report-dashboard' element={<ReportDashboard />} />
+          <Route exact path='/' element={<LandingPage />} />
+          <Route exact path='/about-us' element={<AboutUs />} />
```
Comment (mohamdlog) [inline frontend/src/App.js:25]: This is good practice

**Review by mohamdlog — APPROVED** [review]
Looks good, great work.

**Pull request merged**